### PR TITLE
chore(ci): Add SOURCE_REPO env

### PR DESCRIPTION
**Pull request by universal-itengineer**
## Description

- Add SOURCE_REPO. It is git repo prefix for building 3-rd party images.
- pcre builder not needed anymore.

## Why do we need it, and what problem does it solve?

For easy creation of third-party images depending on the environment

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

### CODE REVIEW

File: Taskfile.yaml [hunk 1]
```diff
@@ -26,6 +26,7 @@ vars:
   TRIVY_VERSION: 0.55.0
   target: ""
   VALIDATION_FILES: "tools/validation/{main,messages,diff,no_cyrillic,doc_changes,copyright}.go"
+  SOURCE_REPO: '{{ .SOURCE_REPO | default "https://github.com" }}'
```
Comment (hardcoretime) [inline Taskfile.yaml:29]: Can we get situation when we should have to use tools from different sources?

**Review by hardcoretime — APPROVED** [review]

File: Taskfile.yaml (hunk 1 above)
Comment (diafour) [reply Taskfile.yaml:30]: There is such situation with wget:
https://github.com/deckhouse/virtualization/pull/433/files#diff-8a7c2a335ed9a0f8ae5a03f8d901ac7db1c132af77b70f4f8b4151d68560c474R225

**Comment by diafour** [comment]
Description is not accurate:

> It is git repo prefix for building 3-rd party images.

SOURCE_REPO is used to download not only sources, but also to download binaries. See trivy, yq, etc.

File: images/cdi-artifact/Taskfile.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: "3"
 silent: true
 
 env:
-  CDI_REPO: "https://github.com/kubevirt/containerized-data-importer.git"
+  CDI_REPO: https://github.com/kubevirt/containerized-data-importer.git"
```
Comment (diafour) [inline images/cdi-artifact/Taskfile.yaml:6]:
```suggestion
  CDI_REPO: "https://github.com/kubevirt/containerized-data-importer.git"
```

Missing quote =)

**Review by diafour — APPROVED** [review]

**Pull request merged**